.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- [ ] As requested by the previous reviewer please make `test.md file with the text "This file should be ignored by git".` and  make sure this file should be ignored by git.